core/src/test/java/hudson/UtilTest.java

```diff
@@ -158,6 +158,7 @@ public void testRawEncode() {
             " \"#%/:;<>?", "%20%22%23%25%2F%3A%3B%3C%3E%3F",
             "[\\]^`{|}~", "%5B%5C%5D%5E%60%7B%7C%7D%7E",
             "d\u00E9velopp\u00E9s", "d%C3%A9velopp%C3%A9s",
+            "Foo \uD800\uDF98 Foo", "Foo%20%F0%90%8E%98%20Foo"
```

Comment:
𐎘 U+10398 UGARITIC LETTER THANNA? Heavy.

Reply:
I'm sure it will be used quite frequently in job names :-)